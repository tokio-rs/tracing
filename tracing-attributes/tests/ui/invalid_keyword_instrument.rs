#![allow(unreachable_code)]

#[tracing::instrument(level = "trace", fields(() = "test"))]
fn test_fn() -> &'static str {}

fn main() {}
