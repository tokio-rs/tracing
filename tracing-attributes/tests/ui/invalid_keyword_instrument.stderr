error: expected "field.name" (string literal) or field.name (punctuated identifier)
 --> tests/ui/invalid_keyword_instrument.rs:3:47
  |
3 | #[tracing::instrument(level = "trace", fields(() = "test"))]
  |                                               ^
